vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -278,6 +278,8 @@ class RoomDetailFragment @Inject constructor(
     private lateinit var attachmentsHelper: AttachmentsHelper
     private lateinit var keyboardStateUtils: KeyboardStateUtils
 
+    private var optionsMenu: Menu? = null
```

Comment:
This is a solution to keep ref on `optionsMenu`, but generally we never do that. We just call `invalidateOptionsMenu()` when the state change.